example/js/example.js

```diff
@@ -1,75 +0,0 @@
-
-    // information about the graph
-    var graph = d3.select('#line_graph');
-
-    var width = 900;
-    var height = 500;
-    var margins = {
-        top: 20,
-        bottom: 20,
-        right: 20,
-        left: 100
-    };
-
-    // range functions
-    var xRange = d3.scale.linear().range([margins.left, width - margins.right])
-                .domain([d3.min(secondLineData, function(d) {
-                    return d.x;
-                }), d3.max(secondLineData, function(d) {
-                    return d.x;
-                })]);
-    var yRange = d3.scale.linear().range([height - margins.top, margins.bottom])
-                .domain([d3.min(secondLineData, function(d) {
-                    return d.y
-                }), d3.max(secondLineData, function(d) {
-                    return d.y
-                })]);
-
-    // axis rendering pieces
-    var xAxis = d3.svg.axis()
-                .scale(xRange)
-                .tickSize(5)
-                .tickSubdivide(true);
-    var yAxis = d3.svg.axis()
-                .scale(yRange)
-                .orient('left')
-                .tickSize(5)
-                .tickSubdivide(true);
-    graph.append('svg:g')
-         .attr('class', 'x_axis')
-         .attr('transform', 'translate(0,' + (height - margins.bottom) + ')')
-         .call(xAxis);
-
-    graph.append('svg:g')
-         .attr('class', 'y_axis')
-         .attr('transform', 'translate(' + (margins.left) + ', 0)')
-         .call(yAxis);
-
-    // line charting pieces, the line function is what interpolates the points into the screen pieces
-    var lineFunc = d3.svg.line()
-                   .x(function(d) {
-                    return xRange(d.x);
-                   })
-                   .y(function(d) {
-                    return yRange(d.y);
-                   })
-                   .interpolate('linear');
-
-    graph.append('svg:path')
-         .attr('d', lineFunc(lineData))
-         .attr('stroke', '#09e')
-         .attr('stroke-width', 2)
-         .attr('fill', 'transparent');
-
-    graph.append('svg:path')
-         .attr('d', lineFunc(secondLineData))
-         .attr('stroke', 'green')
-         .attr('stroke-width', 2)
-         .attr('fill', 'transparent');
```

Comment:
This was pre-nugget stuff here that isn't really helpful anymore. Bye bye! :bomb: :boom: